otter/worker/launch_server_v1.py

```diff
@@ -152,9 +152,23 @@ def check_status(server):
         deferred_description=timeout_description)
 
 
-# limit on 1 servers to be created simultaneously
-MAX_CREATE_SERVER = 1
-create_server_sem = DeferredSemaphore(MAX_CREATE_SERVER)
+# single global instance of semaphores
+_semaphores = {}
+
+
+def get_sempahore(operation, conf_name):
```

Comment:
Not sure about Otter, but on Twisted I would definitely say I'd expect to see a `:param:` and perhaps a `:type:` for each parameter, and a `:return:` explaining the return type...